src/_actions/TradeActions.js

```diff
@@ -61,10 +61,24 @@ export const setQuickTradeField = (symbol, tradeType, field, value) => ({
     value,
 });
 
-export const createTrade = symbol => ({
-    type: types.CREATE_TRADE,
-    symbol,
-});
+export const createTrade = symbol =>
+    (dispatch, getState) => {
+        const activeIdx = getState().workspace.get('activeTradeIndex');
```

Comment:
Not liking the super strong dependency of the action to the whole state structure.
Possibly all stuff you get from the store can be passed as parameters?